code/game/objects/items/storage/medkit.dm

```diff
@@ -375,6 +375,7 @@
 		/obj/item/storage/box/evilmeds = 1,
 		/obj/item/reagent_containers/medigel/sterilizine = 1,
 		/obj/item/clothing/glasses/hud/health/night/science = 1,
+		/obj/item/organ/internal/cyberimp/cyberlink/nt_high = 1, //Monkestation edti: unable to use the hacked surgery toolset without this
```

Comment:
there should be a syndicate version you can give them

Reply:
There is but it does not have the correct programming to the surgery toolset

Reply:
so you would have to hack it first, which can suck if a nukeops hasnt hacked before and installs the arm before the cyberlink and then realizes they cant use it and has to uninstall the arm again to hack it